model/client4.go

```diff
@@ -8110,3 +8110,16 @@ func (c *Client4) GetPostsUsage() (*PostsUsage, *Response, error) {
 	err = json.NewDecoder(r.Body).Decode(&usage)
 	return usage, BuildResponse(r), err
 }
+
+// GetIntegrationsUsage returns usage information on integrations, including the count of enabled integrations
```

Comment:
not clear, is `including` exclusively enabled integrations or does "usage information" also contains other info being returned?